truss/templates/server/model_wrapper.py

```diff
@@ -19,6 +19,10 @@ def __init__(self, config: dict):
         self._model = None
 
     def load(self):
+        if 'bundled_packages_dir' in self._config:
+            bundled_packages_path = Path('/packages')
+            if bundled_packages_path.exists():
+                sys.path.append(str(bundled_packages_path))
```

Comment:
Not related directly to this PR, but I thought we also appended the model path to `sys.path`. Am I remembering wrong?

Reply:
That's the default structure I believe, /app is already in sys.path, perhaps because the server is started from there.